compiler/src/Language/Bond/Util.hs

```diff
@@ -70,6 +71,11 @@ sepBy s f (x:xs)
 optional :: (Monoid m) => (a -> m) -> Maybe a -> m
 optional = maybe mempty
 
+-- takes a predicate and two items and outputs one based on predicate
```

Comment:
Minor: this doesn't take a predicate. That would be more like `(b -> Bool) -> a -> a`.
Maybe just "if/then/else as a function"?

Reply:
Updated to: `if-then-else as a function`